src/gitingest/cli.py

```diff
@@ -1,10 +1,15 @@
 import os
 import pathlib
 import click
+import sys
+from .encoding import setup_encoding
 
-from gitingest.ingest import ingest
-from gitingest.ingest_from_query import MAX_FILE_SIZE
-from gitingest.parse_query import DEFAULT_IGNORE_PATTERNS
+# Setup encoding first
+setup_encoding()
+
+# Define constants
+MAX_FILE_SIZE = 51200  # 50KB by default
```

Comment:
I think the way Constants are handled right now is messy, not your fault but I'll try to figure out a cleaner way than having constants defined at the top of various files